<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_tela">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>221</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Droid Sans Armenian</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ENVIO PROGRAMADO</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="digitarInfo">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>271</width>
       <height>36</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="programar">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>290</y>
       <width>111</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>programar </string>
     </property>
    </widget>
    <widget class="QPushButton" name="voltarMostrarTela">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Quantidade </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>66</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Section</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>160</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="qntdTela">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="section">
        <item>
         <property name="text">
          <string>Home</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>World</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Local</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Business</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Technology</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="frequenciaEmails">
        <item>
         <property name="text">
          <string>Segunda-feira</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Terça-feira</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quarta-feira</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quinta-feira</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quinta-feira</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sexta-feira</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sábado</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Domingo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Todos os dias</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>frequência </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>dos envios</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testarEvios">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>390</y>
       <width>88</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Testar envios</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
